builder/builder.go

```diff
@@ -1274,13 +1272,23 @@ func (b *Builder) ValidateBundles(bundles []string, lvl ValidationLevel) error {
 // UpdateMixVer automatically bumps the mixversion file +10 to prepare for the next build
 // without requiring user intervention. This makes the flow slightly more automatable.
 func (b *Builder) UpdateMixVer() error {
-	// Deprecate '.mixversion' --> 'mixversion'
-	if _, err := os.Stat(filepath.Join(b.Config.Builder.VersionPath, ".mixversion")); err == nil {
-		b.MixVerFile = ".mixversion"
-		fmt.Println("Warning: '.mixversion' has been deprecated. Please rename file to 'mixversion'")
+	mixVer, _ := strconv.Atoi(b.Config.Builder.MixVer)
+	b.Config.Builder.MixVer = strconv.Itoa(mixVer + 10)
+
+	if !config.UseNewConfig {
+		// Deprecate '.mixversion' --> 'mixversion'
```

Comment:
I've noticed that these deprecation codepaths only exist under the `!config.UseNewConfig` guard, when users go to `--new-config` we can expect them not to need the deprecation warning.

But what about when we get rid of `--new-config`? Is that the end of the deprecation period and we will stop doing those checks? Or will we have to re-introduce the deprecation codepaths to the regular codepath?

Reply:
When we remove --new-config, is this file exists there it will just be ignored. If by then the user hasn' t upgraded to the new format, that will be the least of his problems. All the rest will also not work.

Reply:
I think I'm missing something. The deprecation warnings @matthewrsj is talking about are for people using the old names for the version files (e.g., `.clearurl` -> `upstreamurl`). Are we having a separate deprecation period for people using the version files at all?

I haven't read this whole PR thoroughly yet, but am I right that (if you're using `--new-config`) these files are just ignored altogether immediately? Should we have a period of time where, if these files exist, a warning is output, but the value in them is still used?